core/spec/spec_helper.rb

```diff
@@ -55,4 +55,21 @@
 
   config.include FactoryGirl::Syntax::Methods
   config.include Spree::TestingSupport::Preferences
+
+  config.fail_fast = ENV['FAIL_FAST'] || false
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.